src/const.ts

```diff
@@ -68,18 +68,25 @@ export const DEFAULT_CONSTANT = {
   messageInputControls: {
     blockWhileBotResponding: 10000,
   },
+  tools: {
+    functionCall: {
+      carouselAdapter({ response }) {
+        if (isMealsResponse(response)) {
+          return response.meals.map((it) => ({ title: it.strMeal, url: '', featured_image: it.strMealThumb }));
```

Comment:
can you explain either in ticket or in PR what this meals is?

Reply:
This logic makes carouselAdapter is only for Noom usecase. Can it be used in other cases in the future? If so, we should make carouselAdapter more generic somehow.

Reply:
nvm I think its fine.

Reply:
Yes, that is an initial value. It can be overridden and will be removed.